flink-runtime/src/test/java/org/apache/flink/runtime/checkpoint/DefaultCheckpointPlanCalculatorTest.java

```diff
@@ -198,10 +194,9 @@ private void runWithNotRunningTask(
                                 + " state");
             } catch (ExecutionException e) {
                 Throwable cause = e.getCause();
-                assertThat(cause, instanceOf(CheckpointException.class));
-                assertEquals(
-                        CheckpointFailureReason.NOT_ALL_REQUIRED_TASKS_RUNNING,
-                        ((CheckpointException) cause).getCheckpointFailureReason());
+                assertThat(cause).isInstanceOf(CheckpointException.class);
+                assertThat(((CheckpointException) cause).getCheckpointFailureReason())
+                        .isEqualTo(CheckpointFailureReason.NOT_ALL_REQUIRED_TASKS_RUNNING);
```

Comment:
```suggestion
              assertThatFuture(checkpointPlanCalculator.calculateCheckpointPlan())
                    .withFailMessage("The computation should fail since some tasks to trigger are in %s state", notRunningState)
                    .eventuallyFailsWith(ExecutionException.class)
                    .havingCause()
                    .isInstanceOfSatisfying(CheckpointException.class,
                            e ->
                                   assertThat(e.getCheckpointFailureReason())
                                            .isEqualTo(CheckpointFailureReason.NOT_ALL_REQUIRED_TASKS_RUNNING));
```